certified-connectors/Dokobit Portal/apiDefinition.swagger.json

```diff
@@ -1384,6 +1382,147 @@
         ]
       }
     },
+    "/signing/{token}/update.json": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@epelanis please give a proper description for your connector responses. for example you can use "200", "OK", "Sucess", "Bad Request", etc.
Please fix all the responses in the swagger file.

Reply:
Fixed